StortSpelProjekt/Game/src/Gamefiles/EnemyFactory.cpp

```diff
@@ -199,6 +200,11 @@ Entity* EnemyFactory::Add(const std::string& entityName, EnemyComps* comps)
 		ae->AddVoice(comps->OnGruntSounds[0], L"OnGrunt");
 	}
 
+	if (comps->isSpider)
```

Comment:
Dont like this, I would like to see an entry in EnemyComps instead to avoid if statements.